### PR TITLE
updates

### PR DESCRIPTION
**Pull request by j3nsykes**
JSON data structure updates

### CODE REVIEW

File: examples/HTTP/DenseCap/DenseCap_012.pde [hunk 1]
```diff
@@ -53,7 +53,7 @@ void setup() {
   // disable automatic polling: request data manually when a new frame is ready
   runway.setAutoUpdate(false);
   // setup camera
-  camera = new Capture(this,640,480);
+  camera = new Capture(this, 640, 480);
```
Comment (orgicus) [inline examples/HTTP/DenseCap/DenseCap_012.pde]: the spaces are great: makes the code easier to read

File: examples/HTTP/DenseCap/DenseCap_012.pde [hunk 2]
```diff
@@ -81,47 +81,57 @@ void draw() {
...
-    
-    String className = result.getString("class");
-    float score = result.getFloat("score");
-    JSONArray box = result.getJSONArray("bbox");
-    // extract values from the float array
-    float x = box.getFloat(0);
-    float y = box.getFloat(1);
-    float w = box.getFloat(2);
-    float h = box.getFloat(3);
+  for (int i = 0; i < data.size(); i++) {
```
Comment (orgicus) [inline examples/HTTP/DenseCap/DenseCap_012.pde]: JSON update in the DenseCap example is great

File: examples/HTTP/GPT2/GPT2_012.pde [hunk 3]
```diff
@@ -107,5 +107,5 @@ void runwayDataEvent(JSONObject runwayData){
   // point the sketch data to the Runway incoming data 
   data = runwayData;
   //get the value of the "text" key
-  text_output = data.getString("text");
+  text_output = data.getString("generated_text");
```
Comment (orgicus) [inline examples/HTTP/GPT2/GPT2_012.pde]: JSON update in the GPT example is great

File: examples/HTTP/DenseCap/DenseCap_012.pde [hunk 4]
```diff
@@ -81,47 +81,57 @@ void draw() {
...
+
     // display bounding boxes
     noFill();
-    rect(x,y,w,h);
+    rect(x, y, w, h);
     fill(255);
-    text(className + " score: " + String.format("%.2f",score),x,y);
+    text(label + " score: " + String.format("%.2f", val), x, y);
+
+//console logs
```
Comment (orgicus) [inline examples/HTTP/DenseCap/DenseCap_012.pde]: are these console logs still needed ?
(if not, please remove)

File: examples/HTTP/GPT2/GPT2_012.pde [hunk 5]
```diff
@@ -107,5 +107,5 @@ void runwayDataEvent(JSONObject runwayData){
   // point the sketch data to the Runway incoming data
```
Comment (orgicus) [inline examples/HTTP/GPT2/GPT2_012.pde]: Last remarks:

- please remove the .DS_Store file (I should've added that to a .gitignore, my bad)
- have the sketches changed names ? (e.g. GPT2.pde -> GPT2_012.pde  / DenseCap.pde -> DenseCap_012.pde ?) if so, can they be renamed without the _012 suffix ?

Thank you so much for the contribution and apologies for the late reponse.
I will have another look after you commit the changes and merge 🙌

**Review by orgicus — CHANGES_REQUESTED** [review]
Hi Jen,

Thank you for the update: this looks great !

The are only a few nitpicking minor changes required.
Hope that's ok with you.
I'll review again once I spot your new commit addressing these.

Yaaaay!

**Comment by j3nsykes** [comment]
No problem, I'll get these changes made!

**Comment by j3nsykes** [comment]
Changes made. :)

**Pull request merged**